lib/internal/Magento/Framework/HTTP/PhpEnvironment/Request.php

```diff
@@ -696,7 +696,7 @@ public function getHeader($name, $default = false)
         if ($header instanceof HeaderInterface) {
             return $header->getFieldValue();
         }
-        return false;
+        return $header;
```

Comment:
Let's return the default value, as described in the PR.
```suggestion
        return $default;
```

Reply:
@ihor-sviziev Default value set as parameter to parent call https://github.com/magento/magento2/pull/32520/files#diff-606a62cd9ef8cebf42721c86f988cac473eb42ee9947446790d60517e7fdf7d4L695

Reply:
Yes, you're right. But that's not obvious at all. To prevent introducing new issues here in the future, that's why I suggested changing it. Does it make sense?